Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -1902,7 +1934,10 @@ Codeunit 4037 "Helper Functions"
         if not VendorEFTBankAccountsCreated() then
             CreateVendorEFTBankAccounts();
 
-        if CheckBooksCreated() and OpenPurchaseOrdersCreated() and FiscalPeriodsCreated() and VendorEFTBankAccountsCreated() then
+        if not CustomerClassesCreated() then
+            CreateCustomerClasses();
+
+        if CheckBooksCreated() and OpenPurchaseOrdersCreated() and FiscalPeriodsCreated() and VendorEFTBankAccountsCreated() and CustomerClassesCreated() then
```

Comment:
We should try to come up with a better statement than just adding to the IF statement as we continue to add stuff here. Not sure what that might be just yet but something to think about.

Reply:
I was thinking about this too, it is getting pretty long. It might make sense to move that logic to the Configuration table. I'm going to commit this change and see what you think about it.

Reply:
I think I like it.